README.md

```diff
@@ -71,6 +71,34 @@ dq deploy /path/to/config.json
 
 Deploys this Data Quality repository to a remote.
 
+### Generate
+
+```
+dq generate GeneratorClass http://endpoint_to_data_sources /path/to/config/file
+```
+
+There is currently one built-in generator for CKAN instances.
+Ex: Generating a database from `data.gld.gov.au`:
+
+```
+dq generate CkanGenerator https://data.qld.gov.au/ tests/fixtures/dq.json
```

Comment:
@georgiana-b 

If we are reaching into the test directory for a config file on basic functionality, something is wrong :).

We need to either (or both):

1. Have the default config settings in the CLI, and the config file only overrides these settings, so it works in some basic manner without the config file.
2. Have the default config in the root of the project, shipping with the CLI itself.